common.mk

```diff
@@ -364,6 +364,12 @@ buildroot-cleaner:
 ################################################################################
 LINUX_COMMON_FLAGS ?= LOCALVERSION= CROSS_COMPILE=$(CROSS_COMPILE_NS_KERNEL)
 
+.PHONY: linux-menuconfig-common
+linux-menuconfig-common: linux-defconfig
+	$(MAKE) -C $(LINUX_PATH) \
+		$(filter-out Image script_gdb uImage zImage,\
```

Comment:
Why do we have to filter-out these? Wouldn't just calling menuconfig work? In any case.

`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

Reply:
The kernel would be built first before running "make menuconfig" so it's a bit inefficient, especially since we're likely to rebuild right after.

Reply:
TBH none of `Image script_gdb uImage zImage` should be in `$(LINUX_COMMON_FLAGS)` because they are targets, not flags. Some small refactoring would be in order here I believe :wink:

Reply:
Makes sense, I'll update this PR.